docs/source/use_cases/linkage_disequilibrium/compute_linkage_disequilibrium.rst

```diff
@@ -0,0 +1,70 @@
+Compute Linkage Disequilibrium on a Variant Set
+===============================================
+
+.. comment: begin: goto-read-the-docs
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/linkage_disequilibrium/compute_linkage_disequilibrium.html
+
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+.. toctree::
+   :maxdepth: 2
+
+.. contents::
+
+This pipeline calculates linkage disequilibrium between pairs of variants in a Variant Set. It takes as input a VariantSet for which the linkage disequilibrium values will be calculated and calculates the D' and allelic correlation measures of linkage disequilibrium, defined in
```

Comment:
add link on VariantSet to https://cloud.google.com/genomics/reference/rest/v1/variantsets 

in a Global Alliance VariantSet (which you can create from a VCF file https://cloud.google.com/genomics/v1/load-variants).